cabal-testsuite/PackageTests/BuildToolDependsNonComponent/setup.test.hs

```diff
@@ -0,0 +1,4 @@
+import Test.Cabal.Prelude
+-- Test build-tool-depends between two packages
```

Comment:
Could you name the test file `cabal.test.hs`, as it's testing `cabal-install` not `Setup`.

Also we have issue tests as e.g. `./PackageTests/Regression/T4202/cabal.test.hs` but I have no strong opinion on that.

Reply:
I will rename the test file and move it to issue tests.